src/state/account/auth.ts

```diff
@@ -59,4 +63,7 @@ export const refreshToken = async () => {
   Cookies.set('atoken', atoken, COOKIES_PARAMS);
 };
 
-export const getUserEmail = () => jwt(Cookies.get('atoken')).email;
+export const getUserEmail = () => {
+  const token = getToken();
+  return token === undefined ? undefined : jwt<AccessToken>(token).email;
```

Comment:
question: should you check if `typeof token === 'undefined'` or is that not appropriate in TypeScript?

Reply:
i think maybe that is not even necessary in javascript since es2015?